torch/distributed/checkpoint/filesystem.py

```diff
@@ -654,7 +654,11 @@ def read_data(self, plan: LoadPlan, planner: LoadPlanner) -> Future[None]:
                     else:
                         tensor = cast(
                             Tensor,
-                            torch.load(cast(IO[bytes], file_slice), map_location="cpu"),
+                            torch.load(
+                                cast(IO[bytes], file_slice),
+                                map_location="cpu",
+                                weights_only=False
```

Comment:
Here we can set True because we know it has to be a tensor.
```suggestion
                                weights_only=True
```